playbooks/roles/certbot/tasks/main.yml

```diff
@@ -29,14 +29,21 @@
     port: 80
     proto: tcp
 
-- name: add certbot PPA
-  apt_repository:
-    repo: ppa:certbot/certbot
+- name: Remove certbot apt package if installed
+  apt:
+    name: "{{ item }}"
+    state: absent
+  with_items:
+    - python-certbot-nginx
```

Comment:
@mtyaka, do we know for sure that the functionality provided by this package is either included in the `certbot` snap package or that we don't need/use it?

Reply:
That package contains scripts that will automatically update nginx configuration files, which we don't use.